doc-2200: Appzi PR for Over-Commit Resources to Enhance VM Perf for memory (#7930)
spectrocloud/librarium: docs/docs-content/vm-management/create-manage-vm/advanced-topics/vm-oversubscription.md

### Prerequisites

- An active VMO cluster in Palette.

### Procedure

> You can make several changes to reduce the memory footprint and over-commit the per-VirtualMachineInstance (VMI) memory

Reviewer: ```suggestion
Palette provides several configurations to help you reduce the memory footprint and over-commit the per-VirtualMachineInstance (VMI) memory
```

Author: These aren't Palette configurations but KubeVirt: https://kubevirt.io/user-guide/compute/node_overcommit/ . Unless I'm mistaken.

overhead.

> Log in to [Palette](https://console.spectrocloud.com).

Reviewer: ```suggestion
1. Log in to [Palette](https://console.spectrocloud.com).
```

2. From the left **Main Menu**, click on **Profiles**.

> 3. Select the profile you use to create the cluster with the VMO pack.

Reviewer: ```suggestion
3. Select the cluster profile you used to create your VMO cluster.
```
